openmc/statepoint.py

```diff
@@ -451,7 +451,7 @@ def summary(self):
 
     @property
     def with_summary(self):
-        return False if self.summary is None else True
+        return False if self.summary is False else True
```

Comment:
I just removed it since we can live with `StatePoint.summary` in place of both properties.